### DISCUSSION TITLE
ESLint as a task for Azure DevOps

### DISCUSSION THREAD

**Discussion by bmvalente** [eslint/eslint#14773] [Ideas]
By enabling ESLint to be used in the pipelines, we could ensure that the code would get continuously scanned.

**Comment by nzakas** [comment]
I’m not overly familiar with Azure, but since ESLint is a command line tool, can’t you use it in pipelines already?

**Reply by bmvalente** [reply]
Hi @nzakas. It's true, you can use a cmd line, but before you can use it, you need to download the latest version to the agent and after running it, you need to store and upload the results again to ADO. Usually, a task provides a wrapper for all those steps (and more if needed), hence that was the initial though for my ask.

**Reply by nzakas** [reply]
Ah I see. We don’t have the bandwidth to develop and maintain such a thing. We usually let the community develop plugins for wherever they want to use ESLint.
